src/modules/manage-team/sections/invite/CreateInvite.tsx

```diff
@@ -76,14 +78,27 @@ export class CreateInvite extends Component<Props, State> {
     });
   }
 
+  componentDidMount(): void {
+    this._isMounted = true;
+  }
+
+  componentWillUnmount(): void {
+    this._isMounted = false;
+  }
```

Comment:
Needed some extra mounting administration to prevent state changes after unmount (happens in the tests). This can be fixed when this component is turned into a functional component